app/views/layouts/_header.html.erb

```diff
@@ -10,21 +10,21 @@
         <div class="dropdown">
           <button type="button" class="btn btn-primary dropdown-toggle m-1" data-toggle="dropdown">
             <%= image_tag current_user.image.url(:thumb), class: 'zoom' %>
-            <%= full_name %>
+            <%= current_user.full_name %>
           </button>
           <div class="dropdown-menu">
             <%= link_to('Logout', destroy_user_session_path, method: :delete) %>
           </div>
         </div>
         <div class="dropdown">
           <button type="button" class="btn btn-primary dropdown-toggle m-1" data-toggle="dropdown">
-          <span data-feather="settings"></span>    
+          <i class="fas fa-cog"></i>    
           </button>
           <div class="dropdown-menu">
-            <% if can? :manage, @user_setting %>
+            <% if can? :edit, @user_setting %>
               <%= link_to "My Settings", edit_user_setting_path(current_user.user_setting) %><br>
```

Comment:
when you have @user_settings instance then why not use it directly instead of current_user.user_setting ?